### PR TITLE
add support for passing module names

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
enables `scan(__name__)`

### CODE REVIEW

File: importscan/tests/test_importscan.py [hunk 1]
```diff
@@ -174,3 +174,7 @@ def test_module_in_zipped():
     scan(moduleinzipped)
 
     assert fixtures.calls == 1
+
+
+def test_scan_from_name():
+    scan('email')
```
Comment (faassen) [inline importscan/tests/test_importscan.py:180]: This test doesn't seem to verify anything in particular. We should check whether it's been touched through import at least.

File: importscan/scan.py [hunk 2]
```diff
@@ -89,6 +90,11 @@ def handle_error(name, e):
       ``handle_error`` does not re-raise the error, the error is
       suppressed.
     """
+    # module name instead of package object
+    if not hasattr(package, '__name__'):
```
Comment (faassen) [inline importscan/scan.py:94]: Could you explain why you want to support string names as an argument?

**Review by faassen — COMMENTED** [review]
I'm curious what the use case is for this one. I'm generally not a big fan of accepting a range of argument types so I'm curious.

File: importscan/tests/test_importscan.py (hunk 1 above)
Comment (faassen) [reply importscan/tests/test_importscan.py:180]: Oh and I'd like to see a test for dotted names too, or is that not supported?

File: importscan/scan.py (hunk 2 above)
Comment (RonnyPfannschmidt) [reply importscan/scan.py:94]: @faassen simply to enable simple usage of `scan(__name__)` which is quite gruesome to write otherwise

**Pull request closed without merging**